model/database/registry.yaml

```
@@ -127,238 +127,181 @@ groups:
-              stability: experimental
+              value: "geode"  # trademark on 'geode'
+              brief: "[Apache Geode](https://geode.apache.org/)"
+              stability: development
             - id: h2
-              value: "h2"
-              brief: "H2"
-              stability: experimental
-            - id: hanadb
-              value: "hanadb"
-              brief: "SAP HANA"
-              stability: experimental
+              value: "h2"  # no trademark
+              brief: "[H2 Database](https://h2database.com/)"
+              stability: development
             - id: hbase
-              value: "hbase"
-              brief: "Apache HBase"
-              stability: experimental
+              value: "hbase"  # trademark on 'hbase'
+              brief: "[Apache HBase](https://hbase.apache.org/)"
+              stability: development
             - id: hive
-              value: "hive"
-              brief: "Apache Hive"
-              stability: experimental
+              value: "hive"  # trademark on 'hive'
+              brief: "[Apache Hive](https://hive.apache.org/)"
+              stability: development
             - id: hsqldb
-              value: "hsqldb"
-              brief: "HyperSQL DataBase"
-              stability: experimental
+              value: "hsqldb"  # no trademark
+              brief: "[HyperSQL Database](https://hsqldb.org/)"
+              stability: development
+            - id: db2
+              value: "ibm.db2"  # trademark on 'db2'
+              brief: "[IBM Db2](https://www.ibm.com/db2)"
+              stability: development
+            - id: ibm.informix
+              value: "ibm.informix"  # trademark on 'informix'
+              brief: "[IBM Informix](https://www.ibm.com/products/informix)"
+              stability: development
+            - id: ibm.netezza
+              value: "ibm.netezza"
+              brief: "[IBM Netezza](https://www.ibm.com/products/netezza)"  # trademark on 'netezza'
+              stability: development
             - id: influxdb
-              value: "influxdb"
-              brief: "InfluxDB"
-              stability: experimental
-            - id: informix
-              value: "informix"
-              brief: "Informix"
-              stability: experimental
-            - id: ingres
-              value: "ingres"
-              brief: "Ingres"
-              stability: experimental
+              value: "influxdb"  # trademark on 'influxdb'
+              brief: "[InfluxDB](https://www.influxdata.com/)"
+              stability: development
             - id: instantdb
-              value: "instantdb"
-              brief: "InstantDB"
-              stability: experimental
-            - id: interbase
-              value: "interbase"
-              brief: "InterBase"
-              stability: experimental
+              value: "instantdb"  # no trademark
+              brief: "[InstantDB](https://www.instantdb.com/)"
```

from their website, looks like they're calling it just "Instant" (but we could still use `instantdb` as the value based on their domain name

```suggestion
              brief: "[Instant](https://www.instantdb.com/)"
```